smt/surrogate_models/tests/test_surrogate_model_examples.py

```diff
@@ -207,7 +207,115 @@ def test_krg(self):
         plt.ylabel("y")
         plt.legend(["Training data", "Prediction"])
         plt.show()
+    
+    def test_krg_mixed(self):
+        import numpy as np
+        import matplotlib.pyplot as plt
+
+        from smt.surrogate_models import KRG
+
+        xt = np.array([0.0, 1.0, 2.0, 3.0, 4.0])
+        yt = np.array([0.0, 1.0, 1.5, 0.5, 1.0])
+        
+        ##    Vartype example
+    
+        # vartype =["cont","int",("cate",3),("cate",2)]
+        #"cont" means x1 continuous
+        #"int" means x2 integer
+        #"(cate, 3)" means x3,x4 & x5 are 3 levels of the same categorical variable
+        #"(cate, 2)" means x6 & x7 are 2 levels of the same categorical variable
+        
+        vartype = ["int"]     
+        sm = KRG(theta0=[1e-2], vartype=vartype)
+        sm.set_training_values(xt, yt)
+        sm.train()
+
+        num = 100
+        x = np.linspace(0.0, 4.0, num)
+        y = sm.predict_values(x)
+
+        plt.plot(xt, yt, "o")
+        plt.plot(x, y)
+        plt.xlabel("x")
+        plt.ylabel("y")
+        plt.legend(["Training data", "Prediction"])
+        plt.show()
+        
+    
+
+    def test_krg_mixed_2D(self):
```

Comment:
Can I move it to test_krg_based.py ? 
There is no test_kpls.py. I could also create a new test file.

Reply:
If the test is related specifically to KPLS then create test_kpls.py. If you test a "generic" behaviour (meaning using either KRG, KPLS or KPLSK it makes no difference), it could go in test_krg_based.py.

Reply:
Ok, thank you !